kn@0.25.rb

```diff
@@ -0,0 +1,29 @@
+require "fileutils"
+
+class Kn < Formula
```

Comment:
```suggestion
class KnAT025 < Formula
```

This is required as every class must be unique across all formulas in an homebrew repo.

Reply:
We need a test on PRs to catch this, same problem happened in homebrew for plugins

Reply:
Updated, thank you!

Reply:
Ugh, I missed a line in the release instructions: https://github.com/knative/release#homebrew-client

> In kn@${PREV_RELEASE}.rb replace class Kn with class KnAT${PREV_RELEASE_DIGITS}, e.g class KnAT019 for an previous release 0.19.